### ISSUE TITLE
base: libssl1.1 CVE-2020-1971

### ISSUE THREAD

**Issue by cr** [GoogleContainerTools/distroless#655]
#652 fixed `base-debian10`, but it looks like `base:latest` is still affected. Our infra keeps pulling `gcr.io/distroless/base:latest@sha256:ca60828b628c3032b2e79bb8f20868ed5314ec5b8a9272523a9d74cfba18192b` and that one makes our Anchore ci whine pitifully.

**Comment by cr** [comment]
Here's what Anchore has to say:

```
Gate                   Trigger        Detail                                                                                                                                              Status        

vulnerabilities        package        MEDIUM Vulnerability found in os package type (dpkg) - libssl1.1 (CVE-2020-1971 - https://security-tracker.debian.org/tracker/CVE-2020-1971)        warn          

vulnerabilities        package        MEDIUM Vulnerability found in os package type (dpkg) - openssl (CVE-2020-1971 - https://security-tracker.debian.org/tracker/CVE-2020-1971)          warn
```

**Comment by chanseokoh** [comment]
`base` is based on Debian 9 oldstable (Stretch). The Debian folks haven't yet upgraded libssl in the oldstable. (Not sure if they will ever do.) Use `base-debian10`.

**Comment by chanseokoh** [comment]
See

Stretch: https://packages.debian.org/stretch/libssl1.1
Buster: https://packages.debian.org/buster/libssl1.1

**Comment by chanseokoh** [comment]
Looks like Debian upgraded libssl1.1 on Stretch, and our bot picked up the new versions (#656), which I think will fix the CVE on Stretch. The new Distroless images will go live within an hour (if the CI/CD doesn't break).

**Comment by cr** [comment]
Great, thanks for the update!

**Comment by cr** [comment]
FWIW, Anchore is still reporting _CVE-2020-1971_ in both _base_ and _base-debian10_. I didn't get around to investigate whether it's just a broken CVE matcher in Anchore (likely) or the vulnerability is still present in the images (unlikely). Until then I'm assuming that it is the former.

**Comment by chanseokoh** [comment]
Probably it's a false alarm from Anchore. According to [CVE-2020-1971]( https://security-tracker.debian.org/tracker/CVE-2020-1971) is fixed in 1.1.1d-0+deb10u4, which is [what `base-debian10` pulls in](https://github.com/GoogleContainerTools/distroless/blob/e3229eb61f53e89364a071298af9b2326058f0e4/package_bundle_amd64_debian10.versions#L60).
